Gemfile

```diff
@@ -48,12 +48,12 @@ gem 'hypdf', '~> 1.0.10'          # PDFInfoAgent
 gem 'weibo_2', github: 'dsander/weibo_2', branch: 'master'
 
 # GoogleCalendarPublishAgent
-gem "google-api-client", require: 'google/api_client'
+gem "google-api-client", '~> 0.7.1', require: 'google/api_client'
```

Comment:
I was looking into this a little for (https://github.com/cantino/huginn/issues/1874) While latest brings in 'breaking changes', it seems to be a minimal fix.

It's not updated in the migrating notes in master yet, but see:
* https://github.com/google/google-api-ruby-client/issues/548
* https://github.com/google/google-api-ruby-client/pull/541

I changed to the following, and inital tests (aka, installation) seemed like it might work ok:
```
gem "google-api-client", require: 'google/apis'
```